mlrun/feature_store/api.py

```diff
@@ -195,18 +195,39 @@ def get_online_feature_service(
     """initialize and return online feature vector service api,
     returns :py:class:`~mlrun.feature_store.OnlineVectorService`
 
-    example::
+    There are two ways to use the function
+    1. As context manager
+        example::
+
+                with get_online_feature_service(vector_uri) as svc:
+                    resp = svc.get([{"ticker": "GOOG"}, {"ticker": "MSFT"}])
+                    print(resp)
+                    resp = svc.get([{"ticker": "AAPL"}], as_list=True)
+                    print(resp)
+
+            example with imputing::
+
+                with open get_online_feature_service(vector_uri, impute_policy={"*": "$mean", "amount": 0)) as svc:
```

Comment:
```suggestion
                with get_online_feature_service(vector_uri, impute_policy={"*": "$mean", "amount": 0)) as svc:
```
The `open` here is redundant.